src/AutomationFramework/Config.java

```diff
@@ -0,0 +1,85 @@
+package AutomationFramework;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Config {
+	public String FireFoxExePath = "C:\\Downloaded Things\\geckodriver.exe";
+	public String ChromeExePath = "C:\\Downloaded Things\\chromedriver.exe";
+	//public String WebsiteURL = "https://california.demo.collaborativefusion.com/";
+	public String WebsiteURL = "https://www.pulsedemo.org";
+	public String ValidCredFilePath = "C:\\Julie\\6-PULSE\\PULSECredentials.txt";
```

Comment:
Can you put this kind of information into a config file somewhere?

Reply:
Agree with Brian. All hard-coded file & URL paths ought to be in a configuration file